### PR TITLE
Improve docker build

### PR DESCRIPTION
**Pull request by RPS98**
---

## Basic Info

| Info | Please fill out this column |
| ------ | ----------- |
| Issue(s) this addresses   |  |
| ROS2 version tested on | Humble, Galactic |
| Aerial platform tested on | Gazebo, PX4 |

---

## Description of contribution in a few bullet points

* Can build docker image setting the branch and the ROS 2 version

## Future work that may be required in bullet points

* Separate Dockerfiles
* Add develop utilities
* Discuss if add this to the aerostack2 documentation
* Discuss .repos files
* Remove user and use root

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,94 +1,68 @@
...
-    python3-setuptools \
-    python3-pip \
-    python3-venv \
-    software-properties-common \
-    vim \
-    net-tools \
-    iputils-ping \
-    xvfb \
-    curl 
+#create an user with a /home/cvar directory
```
Comment (miferco97) [inline docker/Dockerfile:6]: I dont quite like this, I would use /root/ for the docker as it simplifies some interactions with the container

File: docker/Dockerfile [hunk 2]
```diff
@@ -1,94 +1,68 @@
...
+  apt-get install apt-utils sudo software-properties-common -y && \
+  apt upgrade -y
 
-# NVIDIA_STUFF
-# RUN apt-get update && \
-#     apt-get install -y --no-install-recommends \
-#         nvidia-cuda-toolkit \
-#     && rm -rf /var/lib/apt/lists/* \
-#     && apt-get clean
+## create an user
```
Comment (miferco97) [inline docker/Dockerfile:13]: Here it happens the same

**Comment by miferco97** [comment]
We have now docker images generated from each release, we should document the use of them.

**Comment by RPS98** [comment]
Hi @miferco97,
Another advantage is being able to build a Docker with a specific branch, which is useful for testing in development. We should merge it along with #80

**Comment by miferco97** [comment]
I agree, we shall compile all the requirements in a single issue

**Pull request closed without merging**